### ISSUE TITLE
A11y: Direct link rendered image has no alt-text

### ISSUE THREAD

**Issue by samjewell** [grafana/grafana#75357]
**What happened**:

sharing a panel via "[Direct link rendered image](https://ops.grafana-ops.net/render/d-solo/9ATuWtLnz/accessibility%3a-status-in-grafana-project?orgId=1&from=1687466742069&to=1695242742069&panelId=10&width=1000&height=500&tz=Europe%2FLondon)" leads to an image with no alt-text. Seems to me we should try to put at least something in there, as we don't know how these images might get shared

**What you expected to happen**:

Image should have alt-text

This is an A11y issue:

From @ckbedwell :

> the specific WCAG success criterion it fails to meet is [SC 1.1.1 Non-text content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)

**How to reproduce it (as minimally and precisely as possible)**:

On a panel:
- Kebab menu
   - Share

![image](https://github.com/grafana/grafana-image-renderer/assets/2903904/cb0fcc26-72ec-4c9e-b205-4cc8649456c2)

Then:
- Direct link rendered image

![image](https://github.com/grafana/grafana-image-renderer/assets/2903904/8c870fe0-4599-47e6-8ef0-0534297b6c32)

Generated page is an image with no alt-text

![image](https://github.com/grafana/grafana-image-renderer/assets/2903904/fd1e2d83-a0f2-4d12-ad47-d70cfc4c73b7)

**Anything else we need to know?**:

No

**Environment**:
- ~~Grafana Image Renderer version:~~
- Grafana version: Hosted Grafana
- ~~Installed plugin or remote renderer service:~~
- ~~OS Grafana Image Renderer is installed on:~~
- User OS & Browser: Mac OS, Chrome
- ~~Others:~~
